pkg/cnab/provider/duffle.go

```diff
@@ -34,5 +43,41 @@ func (d *Duffle) newDriver(driverName string) (driver.Driver, error) {
 		configurable.SetConfig(driverCfg)
 	}
 
+	// If docker driver, setup host bind mount for outputs
+	// TODO: separate function/add tests
+	if dockerish, ok := driverImpl.(*duffledriver.DockerDriver); ok {
+		outputsDir, err := d.Config.GetOutputsDir()
+		if err != nil {
+			return nil, errors.Wrap(err, "unable to get outputs directory")
+		}
+
+		// Create outputs sub-directory using the bundle name
+		bundleOutputsDir := filepath.Join(outputsDir, claimName)
+		err = d.FileSystem.MkdirAll(bundleOutputsDir, 0755)
```

Comment:
I think the permissions on that folder may be wrong, because it lets anyone on the machine read outputs created by the bundle. Let's do a sweep on permissions in a separate issue. I'll make one so that we can use a variable for this stuff going forward.

Reply:
I've changed to `os.ModePerm` for now... w/ intentions to follow-up across codebase in issue you've mentioned...